index.html

```diff
@@ -4,9 +4,138 @@
+                    </li>
+                    <li class="nav-link">
+                        <a href="#badge" class="fs-s">Badge</a>
+                    </li>
+                    <li class="nav-link">
+                        <a href="#button" class="fs-s">Button</a>
+                    </li>
+                    <li class="nav-link">
+                        <a href="#card" class="fs-s">Cards</a>
+                    </li>
+                    <li class="nav-link">
+                        <a href="#image" class="fs-s">Image</a>
+                    </li>
+                    <li class="nav-link">
+                        <a href="#input" class="fs-s">Input</a>
+                    </li>
+                </ul>
+            </nav>
+        </aside>
+        
+        <!-- ---------Side Navbar end------------ -->
+        <div class="documentation">
+
+            <!-- ------Getting Started--------- -->
+            <section id="introduction" class="section">
+                <h1 class="fs-l">Introduction</h1>
+                <p class="fs-s "> JODD UI is a free, fast, lightweight framework for building websites.</p>
+                <h2 class="fs-ml">How to Use?</h2>
+                <p class="fs-s ">Copy Paste the link to your href in <span class="highlight">&lt;link&gt;</span> before all other stylesheets.</p>
+                
+                <pre class="code-container">
+                    <code class="html fs-s">
+<link rel="stylesheet" href="https://jodd-component-css.netlify.app/main.css">                       
+                    </code>
+                </pre>
+            </section>
+
+            <!-- -------- Alert----------- -->
+            <section class="section" id="alert">
+                <h2 class="fs-ml">Alert</h2>
+                <p class="fs-s">There are four variants of the alert component <span class="highlight">alert-danger</span>, <span class="highlight">alert-success</span>, <span class="highlight">alert-warning</span>, <span class="highlight">alert-info</span>.</p>
+        
+                <div class="alert alert-success">
+                    <p class="fs-s">A success alert</p>
+                </div>
+                <div class="alert alert-danger">
+                    <p class="fs-s">A danger alert</p>
+                </div>
+                <div class="alert alert-warning">
+                    <p class="fs-s">A warning alert</p>
+                </div>
+                <div class="alert alert-info">
+                    <p class="fs-s">A info alert</p>
+                </div>
+
+                <pre class="code-container">
+                    <code class="html fs-s">
+<div class="alert alert-success">
+    <p class="fs-s">A success alert</p>
+</div>
+
+<div class="alert alert-danger">
+    <p class="fs-s">A danger alert</p>
+</div>
+
+<div class="alert alert-warning">
+    <p class="fs-s">A warning alert</p>
+</div>
+
+<div class="alert alert-info">
```

Comment:
Try being consistent with the indentation. You can use extensions like prettier or beautify for it.

Reply:
This is an issue with highlight. The code that you see to the left is because i want the code snippet to start from the left. If I use proper indentation, the code will start from the centre. Let me know if you have any idea on how to overcome this issue.

Reply:
> This is an issue with highlight. The code that you see to the left is because i want the code snippet to start from the left. If I use proper indentation, the code will start from the centre. Let me know if you have any idea on how to overcome this issue.

You can use the css property white-space: pre on the tags where you want to keep spaces inside.